pages/affiliation-link.vue

```diff
@@ -0,0 +1,387 @@
+            variant="soft"
+          />
+        </UDropdown>
+      </template>
+
+      <div v-if="priceIndexOptions.length" class="px-4 py-5 sm:p-6">
+        <UFormGroup label="Edition">
+          <USelect
+            v-model="priceIndex"
+            :options="priceIndexOptions"
+          />
+        </UFormGroup>
+      </div>
+
+      <UTable :columns="tableColumns" :rows="tableRows">
+        <template #link-data="{ row }">
+          <div class="flex items-center gap-2">
+            <UButton
+              icon="i-heroicons-qr-code"
+              variant="outline"
+              size="xs"
+              @click="selectedPurchaseLink = row"
+            />
+            <UButton
+              icon="i-heroicons-document-duplicate"
+              variant="outline"
+              size="xs"
+              @click="copyLink(row.url || '')"
+            />
+            <UButton
+              class="font-mono break-all"
+              :label="row.url"
+              :to="row.url"
+              color="gray"
+              variant="outline"
+              size="xs"
+              target="_blank"
+            />
+          </div>
+        </template>
+      </UTable>
+    </UCard>
+
+    <UModal v-model="isOpenQRCodeModal">
+      <QRCodeGenerator
+        v-if="selectedPurchaseLink"
+        :data="selectedPurchaseLink.url"
+        :file-name="getQRCodeFilename(selectedPurchaseLink.channel)"
+        :width="500"
+        :height="500"
+      >
+        <template #header>
+          <h3 class="font-bold font-mono">
+            Download QR Code
+          </h3>
+          <UButton
+            icon="i-heroicons-x-mark"
+            color="gray"
+            variant="ghost"
+            @click="isOpenQRCodeModal = false"
+          />
+        </template>
+      </QRCodeGenerator>
+    </UModal>
+  </main>
+</template>
+
+<script setup lang="ts">
+import { AFFILIATION_CHANNELS, LIKE_CO_API } from '~/constant'
+import { useCollectionStore } from '~/stores/collection'
+import { getPurchaseLink } from '~/utils'
+
+const collectionStore = useCollectionStore()
+const route = useRoute()
+const router = useRouter()
+const toast = useToast()
+
+const productIdInput = ref(route.query.product_id as string || '')
+const productId = computed(() => {
+  if (!productIdInput.value) {
+    return ''
+  }
+  const input = productIdInput.value.trim()
+  if (input.startsWith('http')) {
+    const url = new URL(input)
+    const id = url.pathname.split('/').pop()
+    if (id?.startsWith('col_') || id?.startsWith('likenft')) {
+      return id
+    }
+  }
+  return input
+})
+const isCollection = computed(() => productId.value?.startsWith('col_'))
+
+const linkSettings = ref([
+  {
+    name: 'Direct Link (Stripe)',
+    value: 'direct'
+  },
+  {
+    name: 'Use Liker Land Link (Product page)',
+    value: 'liker_land'
+  },
+  {
+    name: 'Custom Link',
+    value: 'custom'
+  }
+])
+const linkSetting = ref(linkSettings.value[0].value)
```

Comment:
`Liker Land Link` might be a safer default, stripe link requires prior knowledge on utm qs etc